packages/compass-crud/src/components/table-view/document-table-view.less

```diff
@@ -61,8 +61,8 @@
     }
 
     &-parent {
-      font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
-      font-size: 11px;
+      font-family: 'Source Code Pro', Menlo, monospace;
```

Comment:
Is `Source Code Pro` a font we load somewhere?

Reply:
Loaded in renderer entrypoint: https://github.com/mongodb-js/compass/blob/53ac5525a645befb1f311193d81f6dd6d17510e2/packages/compass/src/app/index.jsx#L34

Reply:
ty!